src/common/util/cilfacade.ml

```diff
@@ -66,8 +66,32 @@ let parse fileName =
     E.s (E.error "There were parsing errors in %s" fileName_str);
   file
 
+class cleanCilPrinterClass =
```

Comment:
I think it would be good to have an explanation here what the printer does differently than the normal one? I guess not print line numbers and not print `builtins`?